src/etl/impl/SubscriptionSource.hpp

```diff
@@ -103,7 +109,7 @@ class SubscriptionSource {
      * @param onNewLedger The onNewLedger hook. Called when a new ledger is received
      * @param onLedgerClosed The onLedgerClosed hook. Called when the ledger is closed but only if the source is
      * forwarding
-     * @param connectionTimeout The connection timeout. Defaults to 30 seconds
+     * @param wsTimeout A timeout for websocket operations. Defaults to 30 seconds
```

Comment:
We are sure the ledger stream will send us data every ledger. The ledger close time may exceed 30s for some ledger like [89898224](https://livenet.xrpl.org/ledgers/89898224) to [89898225](https://livenet.xrpl.org/ledgers/89898225 ) .  20s gap seems happen from time to time. [89898219](https://livenet.xrpl.org/ledgers/89898219) to [89898220](https://livenet.xrpl.org/ledgers/89898220). We can relax the timeout to 1min maybe.

Reply:
We receive messages from all subscriptions by one websocket connection. I think in validation stream there is no long gaps. But in the worst case we will just reconnect which is quite quick. WDYT?

Reply:
It looks like the validations stream update quick. I don't know if it is guaranteed.  Yes, in the worst case, we reconnect , it seems not much cost. I am okay keep it 30s.